src/utils/helpers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const checkDate = (date) => {
+  const pattern = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/;
+
+  return pattern.test(date);
+};
+
+const validateExpenseBody = (reqBody) => {
+  const { userId, spentAt, title, amount, category, note } = reqBody;
+
+  if (
+    isNaN(userId)
```

Comment:
Avoid using the global isNaN function. It has issues when dealing with non-number values. Consider using Number.isNaN function to make the behavior explicit.